src/plugins/gauge/components/GaugeRadial.vue

```diff
@@ -0,0 +1,401 @@
+<template>
```

Comment:
This component is called GaugeRadial, but it seems to support other gauge types as well? The other gauge types are all defined using templates in this single component, so it feels like they want to be separate components?